content/13.conclusion.md

```diff
@@ -1,9 +1,8 @@
 ## Conclusion {#conclusion}
 
-Collectively, our manuscript is focused on the promotion of practical tips, cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.
-Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these methods can generate innovative biomedical insights, tools, technologies, services, and products.
-A number of our tips (see Tips 7, 8, 9, and 10) are deeply motivated by the need to safeguard against key risks, which include the introduction and perpetuation of bias, over- or mis-interpretation of models, poor generalizability, and the potential for harm unto others.
-However, despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways.
-After all, these techniques have already shown a remarkable ability to meet or exceed the performance of human effort and older algorithms across numerous fields and subdisciplines.
-In sum, we hope this work will encourage the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that think carefully and critically about their usage and deployment of powerful deep learning techniques.
-After all, the initial discernment of the potential scientific and societal consequences of one's work is critical to the advancement of accountable, beneficent, and high-quality science.
+Collectively, we have presented practical tips that emphasize cutting-edge insights and describe evolving professional standards.
+In addition, a number of our points (see Tips 7, 8, 9, and 10) are focused on safeguarding against the risks inherent to data science and/or deep learning.
+These include the over- and mis-interpretation of models, poor generalizability, and the potential to harm others.
+However, we want to strongly emphasize that when used ethically and responsibly, deep learning techniques have the potential to add tremendous value across a diverse range of research and healthcare contexts.
+After all, these techniques have already shown a remarkable capacity to meet or exceed the performance of both humans and traditional algorithms, and very much have the potential to uncover high-impact biomedical insights that drive discovery and innovation.
```

Comment:
```suggestion
After all, these techniques have already shown a remarkable capacity to meet or exceed the performance of both humans and traditional algorithms and have the potential to uncover biomedical insights that drive discovery and innovation.
```
Toning this down a little is still very positive but not quite as over-the-top strong.